onadata/apps/logger/signals.py

```diff
@@ -0,0 +1,19 @@
+# coding: utf-8
+import logging
+from django.db.models.signals import pre_delete
+from django.dispatch import receiver
+
+from onadata.apps.logger.models.attachment import Attachment
+
+
+@receiver(pre_delete, sender=Attachment)
+def post_delete_asset(instance, **kwargs):
+    # Unfortunately, it seems that Django does not call Model.delete() on
+    # delete CASCADE. But this signal is called though.
+    # We want to delete all files when an Instance (or Attachment) object is
+    # deleted.
+    try:
+        instance.media_file.delete()
+    except Exception as e:
+        logger = logging.getLogger('console_logger')
+        logger.error(str(e), exc_info=True)
```

Comment:
i was concerned this would bypass sentry, but it does not. however, i'm not clear that `getLogger('console_logger')` is actually necessary for anything. all it seems to do is prepend `console_logger` to output instead of `root`:
```
In [1]: import logging                                                                                                                                                                       

In [2]: logging.error("what's up?")                                                                                                                                                          
ERROR:root:what's up?

In [3]: logger = logging.getLogger('console_logger')                                                                                                                                         

In [4]: logger.error("console_logger test")                                                                                                                                                  
ERROR 2022-02-22 11:14:43,429 <ipython-input-4-ec0b14870e74> 816 140694592931648 console_logger test
ERROR:console_logger:console_logger test
```
if necessary we could add something like [this](https://github.com/kobotoolbox/kpi/blob/d35fecc3f09d5377da99430428af7cf9b18bdf4a/kpi/utils/log.py) to kobocat, but it does not seem necessary at this time :thinking: